### PR TITLE
syslog: support for systemd sessionId starting with c (fixes #124)

### LINKED ISSUE #124: syslog : systemd-logind sessionid parser
Systemd (Debian 8 here) have `sessionid` starting with `c` plus integer instead of a simple integer. 

```
Apr 27 13:45:01 aragorn sshd[16723]: pam_unix(sshd:session): session opened for user root by (uid=0)
Apr 27 13:45:01 aragorn systemd-logind[61554]: New session c43046 of user root.
Apr 27 13:45:01 aragorn sshd[16723]: Starting session: command for root from 192.168.3.53 port 30874
Apr 27 13:45:01 aragorn sshd[16723]: pam_unix(sshd:session): session closed for user root
Apr 27 13:45:01 aragorn systemd-logind[61554]: Removed session c43046.
```

### PR DESCRIPTION
**Pull request by fvanniere**
I've kept the integer as sessionId.

### CODE REVIEW

File: syslog/modules/lpeg/syslog_message.lua [hunk 1]
```diff
@@ -732,13 +732,13 @@ prog_grammar["sudo"] = l.Ct(
...
                                  l.P"New session "
-                               * l.Cg(integer, "session_id")
+                               * l.P"c"^-1 * l.Cg(integer, "session_id")
                                * l.P" of user "
                                * capture_followed_by("user_id", "." * l.P(-1))
                                * l.Cg(l.Cc("SESSION_START"), "sd_message")
                                ) + (
                                  l.P"Removed session "
-                               * l.Cg(integer, "session_id")
+                               * l.P"c"^-1 * l.Cg(integer, "session_id")
```
Comment (trink) [inline syslog/modules/lpeg/syslog_message.lua]: I realize this file has a bunch of repetition but lets start improving the situation by making this a single session_id grammar used in both places, thanks.

**Comment by trink** [comment]
The change in 3c68015b2e7d44aff304082f8471b4aa73a15156 have already been squash/merged into dev please update your local dev branch to produce a PR with only the syslog change, thanks.

**Comment by sathieu** [comment]
According to source code, the id can be an arbitrary string. It is currently a number optionally prefixed (if console) by c. But changing the type will break ElasticSearch mapping.

@fvanniere Keeping only the integer in session_id is ok for me. Can you also add a flag when "c" is present. Something like l.Cg(l.P"c" , "session_type")^-1 ?

**Comment by trink** [comment]
Replaced by #130

**Pull request closed without merging**